fetch_bringup/launch/fetch.launch

```diff
@@ -2,6 +2,7 @@
   <!-- robot specific information on parameters https://github.com/ros-infrastructure/rep/pull/104 -->
   <param name="robot/type" value="fetch" />
   <param name="robot/name" textfile="/etc/hostname" />
+  <arg name="launch_teleop" default="true" />
```

Comment:
Let's add these changes to freight.launch in freight_bringup, as well, to keep the files similar.  Thanks!

Reply:
I pushed an additional commit.
Thanks!